### PR TITLE
Load Loadscreen images via the virtual filesystem.

### PR DESCRIPTION
**Pull request by pchote**
You all know the drill by now.  This ports load screens to use the VFS in preparation for #10703.

### CODE REVIEW

File: mods/d2k/mod.yaml [hunk 1]
```diff
@@ -126,7 +126,7 @@ Translations:
 	d2k:languages/english.yaml
 
 LoadScreen: LogoStripeLoadScreen
-	Image: d2k:uibits/loadscreen.png
+	Image: ./mods/d2k/uibits/loadscreen.png
```
Comment (Phrohdoh) [inline mods/d2k/mod.yaml]: Why is this being reverted?

Comment (pchote) [inline mods/d2k/mod.yaml]: For the same reason as the last 4 prs.  The FileSystem path resolving code has never supported the custom mod.yaml : shorthand.  #10703 restores the behaviour.

Comment (Phrohdoh) [inline mods/d2k/mod.yaml]: Alright.

**Comment by obrakmann** [comment]
Looks fine as well, :+1:

**Comment by penev92** [comment]
:+1:

**Comment by penev92** [comment]
[Changelog](https://github.com/OpenRA/OpenRA/wiki/Changelog-%28bleed%29/_compare/97ba057f0f27406eb196e69cd2d108bd78757afa)

**Comment by Mailaender** [comment]
:-1: This broke the Red Alert 2 mod https://github.com/OpenRA/ra2/issues/117.

**Comment by pchote** [comment]
Specify the full path until the followup pr is merged.

**Comment by abcdefg30** [comment]
> Specify the full path until the followup pr is merged.

How does this work when the mod is used from the support directory?

**Comment by pchote** [comment]
It won't until #10703.  This point was discussed in almost all the other related prs, so I thought it didn't need to be said again (hence the "You all know the drill by now").

**Comment by abcdefg30** [comment]
I know. I was just curious if there is another secret way to do it though, as actually, (if you had known/) you know that "Specify the full path until the followup pr is merged." won't work for the support dir, why did you give this answer then? :p

**Comment by Mailaender** [comment]
Actually it works with a hard-coded (absolute) full path:

```yaml
LoadScreen: LogoStripeLoadScreen
	Image: /home/matthias/.openra/mods/ra2/uibits/loadscreen.png
```

which I currently use as a workaround.

**Pull request merged**